appLegacyFragment/src/main/res/layout/activity_legacy_fragment.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There's nothing linear here. You can just make the outter most View the container view